CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## TBD
```

Comment:
Do we have a changelog format we want to follow? This deviates slightly from https://keepachangelog.com/en/1.0.0/ . Perhaps this should be `[Unreleased]`? Additionally, should we choose to adopt https://keepachangelog.com/en/1.0.0/, we should keep a link at the top of our changelog to point contributors to a guide of how to modify the changelog